frontend/components/JoinUsComponent/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Heading } from '../shared/Heading';
+import Button from '../shared/Button';
+
+const JoinUs = ({ data }) => {
+  const { description, left_img, right_img } = data;
+  return (
+    <div className="border-4 border-dashed border-c100">
+      <div className="flex justify-around">
+        <div className="my-24 w-1/6">
+          <img
+            className="object-cover w-11/12 hidden md:block"
+            src={left_img}
+            alt=""
+          />
+        </div>
+        <div className="w-3/6 flex flex-col justify-center mb-24">
+          <h1 className=" text-center mt-16 py-5 uppercase text-lg font-black">
```

Comment:
Now you can move all styling to Heading component and use the custom sizes and options of heading component